lang/ui.ca.json

```diff
@@ -0,0 +1,1161 @@
+    "description": "Menu option to delete file"
+  },
+  "deleted-file": {
+    "defaultMessage": "S'ha suprimit {filename}",
+    "description": "Confirmation message that file got deleted"
+  },
+  "disconnect-action": {
+    "defaultMessage": "Desconnecta",
+    "description": "Disconnect button text"
+  },
+  "disconnect-hover": {
+    "defaultMessage": "Desconnecta't de la micro:bit",
+    "description": "Disconnect button hover text"
+  },
+  "dont-show-again": {
+    "defaultMessage": "No tornis a mostrar això",
+    "description": "Text to never show a dialog again"
+  },
+  "drag-hover": {
+    "defaultMessage": "Arrossegar i deixar anar",
+    "description": "Draggable code hover text"
+  },
+  "edit-file-action": {
+    "defaultMessage": "Edita {name}",
+    "description": "Menu option to edit file"
+  },
+  "edit-name-project-hover": {
+    "defaultMessage": "Edita el nom del teu projecte",
+    "description": "Text for user to edit the name of the project"
+  },
+  "edit-project-name-action": {
+    "defaultMessage": "Edita el nom del projecte",
+    "description": "Text for user to edit the name of the project"
+  },
+  "failed-to-build-hex": {
+    "defaultMessage": "No s'ha pogut crear el fitxer hexadecimal",
+    "description": "Error title when we cannot build the hex file, for example because no storage space. Additional detail is shown separately."
+  },
+  "feedback": {
+    "defaultMessage": "Comentaris",
+    "description": "Feedback action"
+  },
+  "file-actions": {
+    "defaultMessage": "Accions del fitxer {name}",
+    "description": "Header of file actions for file"
+  },
+  "file-already-exists": {
+    "defaultMessage": "Aquest fitxer ja existeix",
+    "description": "File already exists notification text"
+  },
+  "file-name-invalid-character": {
+    "defaultMessage": "El nom conté un caràcter no vàlid: {invalid}",
+    "description": "Warning text for new Python file name with an invalid character"
+  },
+  "file-name-length": {
+    "defaultMessage": "El nom és massa llarg",
+    "description": "Warning text for overly long file names (file system limitation)"
+  },
+  "file-name-lowercase-only": {
+    "defaultMessage": "El nom ha de ser tot en minúscula",
+    "description": "Warning text for new Python file with uppercase"
+  },
+  "file-name-not-empty": {
+    "defaultMessage": "El nom no pot estar buit",
+    "description": "Warning text for new Python file with empty name"
+  },
+  "file-name-start-number": {
+    "defaultMessage": "El nom no pot començar amb un nombre",
+    "description": "Warning text for new Python file names starting with a number"
+  },
+  "file-name-whitespace": {
+    "defaultMessage": "El nom no pot contenir espais",
+    "description": "Warning text for new Python file name with whitespace"
+  },
+  "firmware-update-link": {
+    "defaultMessage": "Has d'actualitzar el teu microprogramari <link>update your firmware</link> abans de poder connectar-te a aquesta micro:bit.",
```

Comment:
Translate text in `<link>`, don't duplicate.

Reply:
Requested in Crowdin